Patches/Medical System Expansion/HediffDefs/Hediffs_AddedModules/Patch_Hediffs_AddedModules_ArchotechSpecial.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Checking for the mod ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Medical System Expansion</modName>
+			</li>
+
+			<!-- ==== Special Archotech Modules  ==== -->
+			<!-- MSE_AdvancedRangedPlatformModule -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="MSE_AdvancedRangedPlatformModule"]/stages</xpath>
+				<value>
+					<stages>
+						<li>
+							<minSeverity>0.000001</minSeverity>
+							<capMods>
+								<li>
+									<capacity>Manipulation</capacity>
+									<offset>0.03</offset>
+								</li>
+							</capMods>
+							<statOffsets>
+								<ShootingAccuracyPawn>4</ShootingAccuracyPawn>
+								<AimingAccuracy>0.4</AimingAccuracy>
+								<AimingDelayFactor>-0.2</AimingDelayFactor>
+							</statOffsets>
+						</li>
+					</stages>
+				</value>
+			</li>
+			<!-- MSE_AdvancedMeleePlatformModule -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="MSE_AdvancedMeleePlatformModule"]/stages</xpath>
+				<value>
+					<stages>
+						<li>
+							<minSeverity>0.000001</minSeverity>
+							<capMods>
+								<li>
+									<capacity>Manipulation</capacity>
+									<offset>0.03</offset>
+								</li>
+							</capMods>
+							<statOffsets>
+								<MeleeDodgeChance>0.8</MeleeDodgeChance>
+								<MeleeHitChance>8</MeleeHitChance>
+								<MeleeCritChance>0.8</MeleeCritChance>
+								<MeleeParryChance>0.8</MeleeParryChance>
+							</statOffsets>
+						</li>
+					</stages>
+				</value>
+			</li>
+			<!-- MSE_AdvancedPowerArmModule -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="MSE_AdvancedPowerArmModule"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>fist</label>
+							<capacities>
+								<li>Blunt</li>
+							</capacities>
+							<power>20</power>
+							<cooldownTime>1.95</cooldownTime>
+							<armorPenetrationBlunt>8</armorPenetrationBlunt>
+							<alwaysTreatAsWeapon>true</alwaysTreatAsWeapon>
+						</li>
+					</tools>
+				</value>
+			</li>
+			<!-- MSE_AdvancedPowerClawModule -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="MSE_AdvancedPowerClawModule"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>Claw</label>
+							<capacities>
+								<li>Slash</li>
```

Comment:
Slash is deprecated, use Scratch or Slash.

Reply:
Changed it with 1a53152